app/javascript/helpers/Axios.jsx

```diff
@@ -11,6 +11,7 @@ const axiosInstance = axios.create(
     headers: {
       'Content-Type': 'application/json',
       Accept: 'application/json',
+      'X-CSRF-TOKEN': document.querySelector('[name=csrf-token]').getAttribute('content'),
```

Comment:
What's this doing exactly?

Reply:
It gets the csrf token that Rails sets in th header and passes it to the backend so that Rails can check it on every non GET requests